File: 2.0/python/campaigns/postTTSCampaignSound.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.campaigns.postTTSCampaignSound(
    body={
        'voice': 'MALE1',
        'message': 'This is a TTS sound'
    }
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/campaigns/postTTSCampaignSound.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->postTTSCampaignSound();
        $body = '{
                    "voice":"MALE1",
                    "message":"This is a TTS sound"
                 }';
        $request->getOperationConfig()->setBodyParameter($body);
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
